### PR TITLE
Add EIP: ERC/EIP Repository split

### PR DESCRIPTION
**Pull request by shemnon**
An EIP giving motivation, rational, alternatives, and objections to the EIP/ERC split proposal.

### CODE REVIEW

**Review by ProphetZX — APPROVED** [review]
Kudos for the structure and the great summary of the current pain points of which we spoke during the last office hour meeting. Nothing to add, except that maybe stack might be replaced by chain/ecosystem (line 23, line 28, etc.)?

**Review by xinbenlv — APPROVED** [review]
The text in this PR satisfied the requirement to be merged as a Draft status

File: EIPS/eip-7329.md [hunk 1]
```diff
@@ -0,0 +1,245 @@
+---
+eip: 7329
+title: ERC/EIP Repository split
+description: Split the ERC specifications out of the EIP repository into a new repository, so that only core protocol EIPs remain
+author: Lightclient (@lightclient), Danno Ferrin (@shemnon)
+discussions-to: https://ethereum-magicians.org/t/eip-erc-repo-split/15053
```
Comment (xinbenlv) [inline EIPS/eip-7329.md:6]: Optional: feel free to consider use existing EIP/ERC split FEM thread https://ethereum-magicians.org/t/proposal-forking-ercs-from-eips-repository/12804

@shemnon

Comment (shemnon) [reply EIPS/eip-7329.md:6]: That didn't show up on my searches.  Thanks.

**Comment by gcolvin** [comment]
This got merged despite my block?  Or am I misunderstanding?

**Comment by xinbenlv** [comment]
This is merging to “draft”, doesn’t mean it is being adopted.

I personally still hold my reservation of the content of the proposal, but
I think procedure-wise, the text satisfy to be publish as an EIP Draft.

IIIUC, An Meta EIP being proposed and published as Draft or Review does not
mean it being adopted.
Only moving to Final means the adoption of the proposed EIP, which is what
you are blocking.

Since I am objecting the proposal, I feel that it’s more appropriate for me
to approve it to publish as a draft to show to friendly collaborative
spirit in this debate.

File: EIPS/eip-7329.md [hunk 2]
```diff
@@ -0,0 +1,245 @@
...
+   operation. Once this PR is approved, it should be executed immediately before
+   merging.
+2. The new ERCs repository goes live and includes the changes from the script.
+3. Setup ercs.ethereum.org subdomain and update the CI to point to the ERCs
+   repo.
+4. Set up a redirect for ERCs on eips.ethereum.org to go to the new website.
+5. Create a unified document for editors to assign EIP/ERC numbers. EIPs and
+   ERCs will no longer be based on an initial PR number but on a number
+   incremented by the EIP editors of their respective repositories. EIPs will be
+   assigned even numbers and ERCs will be assigned odd numbers. The exact
```
Comment (mattstam) [inline EIPS/eip-7329.md:65]: typo: need to continue this sentence

**Pull request merged**